doc/ref_cert/README.md

```diff
@@ -1,5 +1,49 @@
 [<< Back](../)
 
+# CNTT Reference Conformance (RC) Requirements and Testing Principles
+
+The objective each RC workstream is to provide an automated mechanism to measure either a network function (NF) or a cloud infrastructure with a standard set of requirements defined by the associated Reference Architecture (RA).  Through this measurement, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
+
+The overall program requires the close coordination of the following:
+
+**Requirements** - The agreed upon rules and recommendations to which a compliant NF or cloud infrastructure must adhere. All requirements will be hosted and maintianed in the RA.
+**Tests** - The verification mechanism that determines that a given NF or cloud infrastructure complies with one or more requirements.
+**Conformance Specifications** - The definition of the requirements, tests, and circumstances (test case integration, etc.) that must be met to be deemed conformant.
+
+If there is not clear traceability and strong links between these 3 components, then it becomes difficult to determine if a NF or cloud infrastructure is compliant. With this in mind, below are the set of recommended principles for each of the three components to follow. Adherence to these principles will provide the following:
+
+* Enable clear progress tracking and linkage between independent projects (i.e. know what has and hasn't been covered, and track changes over time)
+* Help users better understand mandatory vs. optional requirements
+* Provide a stable set of point-in-time requirements and tests to achieve conformance.  
+* Reduce ambiguity in testing, requirements, and conformance
+
+
+## Requirement Principles
+* All requirements must be assigned a requirements ID and not be embedded in narrative text.  This is to ensure that readers do not have to infer if a requirement exists and is applicable
```

Comment:
I'm not clear which level is addressed here. single test or test case? Most of the test frameworks already leverage on single test id (see Tempest) but it's not at the test case level.

Reply:
this should be moved to a higher level where the requirements are held